content/blog/youtrack-github.md

```diff
@@ -0,0 +1,36 @@
+---
+author: "Alex Hill"
+date: 2019-04-17
+title: Integrating YouTrack and GitHub workflows
+best: false
+---
+
+Like most technical teams we use an issue tracker to manage our work. Our tool of choice for
+this is [YouTrack](https://www.jetbrains.com/youtrack/). We also make extensive use of GitHub and to synchronise our 
+workflow across the two platforms we name our git branches with YouTrack ticket ids, so every pull request 
+corresponds to a ticket. This naming convention makes it easier to track work in progress across the two platforms, 
+plus our deploy tool can use git to look at which branches have been merged in and then auto-update the 
+corresponding YouTrack tickets once the code is deployed. 
+
+An issue moves through 5 states before deployment:
+
+1. issue created
+1. issue in progress
+1. review requested
+1. review submitted
+1. issue ready for deploy
+ 
+In the web development team we use an Agile board in YouTrack to track an issue through these states and to know whether 
```

Comment:
People might not have heard of these but would be interested (I showed this to a research group and they were interested) - link to something like https://en.wikipedia.org/wiki/File:Scrum_task_board.jpg ?